x/examples/outline-cli/README.md

```diff
@@ -0,0 +1,30 @@
+# OutlineVPN CLI
+
+The CLI interface of OutlineVPN client for Linux.
+
+## Usage
+
+#### Standard
+
+```
+./outline-cli -transport "ss://<outline-server-access-key>"
+```
+
+#### Advanced (with Golang)
+
+```
+go run github.com/Jigsaw-Code/outline-sdk/x/examples/outline-cli@latest -transport "ss://<outline-server-access-key>"
+```
+
+### Arguments
+
+- `-transport` : the Outline server access key from the service provider, it should start with "ss://"
+
+## Build (for Developers)
+
+We recommend to setup a [go workspace](https://go.dev/blog/get-familiar-with-workspaces) to build the code. Then use the following command to build the CLI (only support Linux):
+
+```
+cd outline-sdk/x/examples/
+go build -o outline-cli  -ldflags="-extldflags=-static" ./outline-cli
```

Comment:
Why do you need `-ldflags="-extldflags=-static`?
Please add a comment.

Reply:
Is that because of the cgo in tun2socks?
As far as I know the cross-compilation should work fine, but perhaps only without cgo.

Reply:
Yes, cgo will pull in the C runtime. And by default they will be linked as dynamic library, which is not working across different Linux versions/distros.

Reply:
Please add a comment to the README explaining that.